### PR TITLE
simplify solution, fix favorite issue, rename tacotruck-solution project

### PR DESCRIPTION
**Pull request by Roland-Hufnagel**
Three in one:
1. Simplify solution (remove useRef)
2. make use of useEffect to fix the issue that bookmarking a spotlight renders a new spotlight
3. css-structure/taco-truck-solution had the same project name like taco-truck, so I renamed it.

### CODE REVIEW

File: sessions/recap-project-5/example-implementation/pages/index.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 export default function SpotlightPage({
   pieces,
   artPiecesInfo,
   onToggleFavorite,
 }) {
-  const spotlightPiece =
-    pieces[Math.floor(Math.random() * pieces.length)];
+  const [spotlightPiece, setSpotlight] = useState(null);
+  useEffect(() => {
+    console.log("useEffekt");
```
Comment (ahohnsen) [inline sessions/recap-project-5/example-implementation/pages/index.js:12]:
```suggestion

```

**Review by ahohnsen — CHANGES_REQUESTED** [review]
Nice! 😎 

Two tiny comments.

File: sessions/recap-project-5/example-implementation/pages/index.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+    setSpotlight(pieces[Math.floor(Math.random() * (pieces.length - 1))]);
```
[the remark this answers is not in the document]
Comment (Roland-Hufnagel) [reply sessions/recap-project-5/example-implementation/pages/index.js:13]: You're absolute right. (nobody saw this detail before 🤔)

**Review by ahohnsen — APPROVED** [review]
Great. 👍🏼

**Pull request merged**